src/ms-store-badge.ts

```diff
@@ -18,6 +18,11 @@ class MSStoreBadge extends HTMLElement {
    */
   cid: string = "";
 
+  /**
+    * Indicates whether app costs money or is free.
+    */
+  paid: boolean = false;
+
```

Comment:
What's the thinking here - is it whether to launch mini PDP or not? 

If so, we should just name it that instead: `mode` with values `mini` and `full` or something.

Also, is this implemented yet (are we doing anything with this field?) If not, let's save it for a separate pull request.

Reply:
Yes, I fixed this! We can save it for a separate pull request when I get it working.